MMM-WeatherChart.js

```diff
@@ -118,18 +122,6 @@ Module.register("MMM-WeatherChart", {
         return params;
     },
 
-    getDayString: function (num) {
```

Comment:
I want to make the displaying of day names as short as possible to avoid overrapping each other in the x-axis. Then, it's difficult to accept this change for now.

Reply:
In German the short names are also two letters. In case in any language it is longer, I cut it now to two letter.

Reply:
Or is it longer in Japanese and therefore you used the English short names?
Then I would add another configuration value to use English or local names.